tendermint/vue
# fix: Add back and title for account dropdown

## Description & Context

After we logged in with Keplr and opened the account menu in the top right corner, we could go to the settings section and there was no way to go back after that. There was also no clear understanding of where we are and decided to display the title of the section. 

## How To Test

1. Open the template;
2. Connect with Keplr;
3. Click on the account name to open the menu;
4. Go to the settings section by clicking on the "Settings" item;
5. You should see a back arrow and a title;
6. When you click outside of the menu and reopen it, the status should be the default;

## Linked issue

Closes #200
